### ISSUE TITLE
Fresh installation of any packages dependent `openapi-client-axios` are broken

### ISSUE THREAD

**Issue by npdev453** [anttiviljami/openapi-client-axios#55]
Hi @anttiviljami, 

yesterday one of my experemental package was broken when was installed freshly (without package-lock.json and node_modules) with next error: 

```
     TypeError: Cannot read property 'dereference' of undefined

      at crawl (node_modules/@apidevtools/json-schema-ref-parser/lib/dereference.js:44:15)
      at Object.dereference [as default] (node_modules/@apidevtools/json-schema-ref-parser/lib/dereference.js:19:22)
      at OpenAPIClientAxios.initSync (node_modules/openapi-client-axios/client.js:206:34)
      at Object.B.default (<index.js>:42:25)
      at new e.B (<index.js>:384:11)
```

After some drilling of some sources I detect that reasons of the problem is in:
* `json-schema-ref-parser` library update from **9.0.6** to **9.0.7** 
[release 9.0.7](https://github.com/APIDevTools/json-schema-ref-parser/releases/tag/v9.0.7 )
["breakin" change](https://github.com/APIDevTools/json-schema-ref-parser/pull/195/files#diff-ecfa1ed906a19d82ad7eb77c547ccf52808aca8ca991ab47f1b337a6dacf4ce3R44) (but not breakin by a docs)

* `openapi-client-axios incorrect` incorrect usage of `derefence` from that library
  `derefence` has a second `options` param that type is [$RefParserOptions](https://github.com/paztis/json-schema-ref-parser/blob/master/lib/dereference.js#L34) and was never marked as optional
  
* `npm install` that does not respect `package-lock.json` of dependencies 
 https://github.com/npm/npm/issues/19458

So, that error can be catched by anyone who will install `openapi-client-axios` in new projects or while force `deps upgrade`,
by that reason asking to fix it asap. 

PR with fix: https://github.com/anttiviljami/openapi-client-axios/pull/54

**Comment by npdev453** [comment]
fix:
https://github.com/anttiviljami/openapi-client-axios/pull/54

**Comment by anttiviljami** [comment]
Hi! I've merged #54 and released `openapi-client-axios@3.7.9`
